src/main/java/main/TessUi.java

```diff
@@ -0,0 +1,124 @@
+package main;
+
+import java.util.Scanner;
+
+/**
+ * Represent the user interface of Tesseract.
+ * @author Fan Jue
+ * @version 0.1.0
+ * @since 0.1.0
+ */
+public class TessUi {
+    // deals with interactions with the user
+    protected static final String INDENT_ONE = "    ";
+    protected static final String INDENT_TWO = "        ";
```

Comment:
For indentation, I would recommend appending "\t" for INDENT_ONE and "\t\t" for INDENT_TWO

Reply:
Wow time to pick up a useful trick~